extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/tracing/intrumentation/vertx/wrapper/OpenTelemetryWrapper.java

```diff
@@ -0,0 +1,11 @@
+package io.quarkus.opentelemetry.runtime.tracing.intrumentation.vertx.wrapper;
+
+import io.opentelemetry.api.OpenTelemetry;
+
+public interface OpenTelemetryWrapper {
+
+    OpenTelemetry get();
+
+    Boolean resetIfChanged();
```

Comment:
Shouldn't this be `boolean` rather than `Boolean`

Reply:
obsolete